app/src/main/res/layout/activity_main.xml

```diff
@@ -49,7 +49,7 @@
             android:id="@+id/pauseButton"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:text="pause"
+            android:text="Pause"
```

Comment:
Changing to capital P, because it looks cleaner.

Reply:
I'd take this two steps further:
1) Extract to strings.xml, so that a) you'll have all of your String resources in one place, and b) better for internationalization, as you can have a separate strings.xml for each language/
2) Use an ImageButton.  Pictures are faster than words.